styles/mixins/button.js

```diff
@@ -1,15 +1,42 @@
+'use strict';
 module.exports = function () {
-  return function (mixin, bgColor, color) {
-    const activeBgColor = 'red';
-    const styles = {};
-    const state = {
-      'background-color': activeBgColor
-    };
-    styles['background-color'] = bgColor;
-    styles.color = color;
+  return function (mixin, bgColor, fgColor, type) {
+
+    const styles = {
+      display: 'inline-block',
+      'line-height': 1,
+      'text-align': 'center',
+      'font-size': '14px',
```

Comment:
shouldn't this be passed in from the config?  Client might have a larger button and the font-size will help determine that.